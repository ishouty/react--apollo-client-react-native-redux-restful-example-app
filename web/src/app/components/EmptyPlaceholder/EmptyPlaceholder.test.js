import React, { Component } from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import EmptyPlaceHolder from './EmptyPlaceHolder'

describe('EmptyPlaceHolder', () => {
  test('Component should render and match snap shot', () => {
    const component = shallow(
      <EmptyPlaceHolder
        imageUrl="/src.jpg"
        helpTextTitle="sampletest"
        helpTextDetail="example test detail"
      />
    )
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
