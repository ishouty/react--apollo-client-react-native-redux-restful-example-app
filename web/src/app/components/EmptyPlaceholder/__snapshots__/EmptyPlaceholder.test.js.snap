// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyPlaceHolder Component should render and match snap shot 1`] = `
<div
  className="empty-placeholder"
>
  <div
    className="img-container"
  >
    <img
      src="/src.jpg"
    />
  </div>
  <div
    className="help-title"
  >
    sampletest
  </div>
  <div
    className="help-detail"
  >
    example test detail
  </div>
</div>
`;
